test/accountlinking/emailpassword.test.js

```diff
@@ -394,7 +394,7 @@ describe(`accountlinkingTests: ${printPath("[test/accountlinking/emailpassword.t
             });
 
             let user = (
-                await EmailPassword.signUp("public", "test@example.com", "password123", {
```

Comment:
add a test case:
discord -> email-otp -> emailverification. Here, the user will have to use their own email for email-otp and not the discord email (in case it's fake), and the email verification claim should pass cause it treats fake emails as verified anyway

Reply:
Added to check list, will add  along with other tests

Reply:
Keeping this open until this test is added.